### ISSUE TITLE
Various translation issues

### ISSUE THREAD

**Issue by Night-Pryanik** [CleverRaven/Cataclysm-DDA#22102]
- [x] "reload_noise" from jsons isn't translated;
![default](https://user-images.githubusercontent.com/11132525/31374856-4f3088c2-ada8-11e7-80a5-5b5d3fa18278.png)
- [x] Reload menu entry sometimes shifts the bars with no obvious reason. In English everything is fine. It seems that "N" translation is lost somehow;
![default](https://user-images.githubusercontent.com/11132525/31374795-0898be98-ada8-11e7-87f6-60bad1651e77.png)
![default](https://user-images.githubusercontent.com/11132525/31375332-d6aa224e-ada9-11e7-8576-49b2ad81536e.png)
- [x] Green default character name header isn't translated;
![default](https://user-images.githubusercontent.com/11132525/31374926-8f00c534-ada8-11e7-948f-047d5c531b83.png)
- [x] Fangs mutation special attack translation is lost somehow;
![default](https://user-images.githubusercontent.com/11132525/31375041-efea75fc-ada8-11e7-866e-a64495059648.png)
- [ ] Translated strings that don't fit in the window seem to lose some symbols;
![default](https://user-images.githubusercontent.com/11132525/31376660-5b6ac520-adae-11e7-99d7-4cc48a8ff927.png)
- [x] Sometimes strings are colored incorrectly;
![default](https://user-images.githubusercontent.com/11132525/31577160-4ca61898-b112-11e7-9e01-85a680e1a0ec.png)
![default](https://user-images.githubusercontent.com/11132525/31577165-5d4b078a-b112-11e7-8875-a85816e2da78.png)
- [x] Attitude is untranslated; status (ellipsis at the right edge) is shifted;
![default](https://user-images.githubusercontent.com/11132525/31577485-0eaacfe2-b118-11e7-9183-a35f00cd9f30.png)
- [x] Line shifting.
![default](https://user-images.githubusercontent.com/11132525/31585244-65258eb6-b1c7-11e7-8355-2a14a241a55b.png)

**Comment by Night-Pryanik** [comment]
Seems to fix the second issue. Had to lurk through the code for it. Directions must have 5 symbols, e.g. above and south = "`UP_S `" (UP_S and one whitespace). There can be no good abbreviation for UP, or DN, or CE in other languages, so for example in Russian we chose to translate "`UP_S`" as "`^Ю`". As you can see, the result has two symbols instead of three symbols in the original text, so we must add missing symbols as whitespaces.

**Comment by AMurkin** [comment]
no. 4 - `extract_mutation` in `extract_json_strings.py` counts `attacks` as objects, but they also can be arrays of objects.

```
"//extracted",
"attacks" : {
    "attack_text_u" : "You bite into %s with your ratlike incisors",
    "attack_text_npc" : "%1$s bites %2$s with their ratlike incisors",
},

"//failed",
"attacks" : [
    {
        "attack_text_u" : "You sink your fangs into %s",
        "attack_text_npc" : "%1$s sinks their fangs into %2$s"
    },
    {
        "attack_text_u" : "You sink your fangs into %s",
        "attack_text_npc" : "%1$s sinks their fangs into %2$s"
    }
]
```

**Comment by AMurkin** [comment]
no. 1 - `"reload_noise"` now handled by `def extract(item, infilename)`, but items with `"type": "GUN"` really handled by `def extract_gun(item)`.
```
if "reload_noise" in item:
    item_reload_noise = item.get("reload_noise")
    writestr(outfile, item_reload_noise)
```
I think.

**Comment by Night-Pryanik** [comment]
Untranslated attitude was fixed in #22623.

**Comment by AMurkin** [comment]
Suggest to test and split remaining issues.

**Comment by Night-Pryanik** [comment]
The remaining issues are still exist on 0.C-7777.

**Comment by kevingranade** [comment]
Last issue supposedly fixed by #25168 but linking string was malformed.
